meeting-minutes.md

```diff
@@ -0,0 +1,11 @@
+# Meeting Minutes
+
+Every Thursday at 9:30 AM Pacific, the Helm team has a meeting to discuss the
+state of the project. This meeting is open to the public.
+
+This document tracks the meeting minutes for those meetings.
+
+Meeting minutes are captured as Google Docs (following SIG methodology) so that
+Helm contributors can collaborate.
+
+- [Aug 17, 2017](https://docs.google.com/document/d/1elWRfvH3AkHdr8pOaqyPbqSZ6ONR-l1Sb9_gapqh8ZA/edit?usp=sharing)
```

Comment:
Any specific reason for creating separate docs per meeting? SIG-Apps, Charts and k8s community meetings all use a single doc (e.g. https://docs.google.com/document/u/1/d/1LZLBGW2wRDwAfdBNHJjFfk9CFoyZPcIYGWU7R1PQ3ng/edit#)